Simplenote/fr.lproj/Localizable.strings

```diff
@@ -47,10 +47,11 @@
 "Bad email or password" = "Adresse e-mail ou mot de passe incorrect";
 
 /* Delete account confirmation alert message */
-"By deleting your account, all notes created with this account will be permanently deleted. This action is not reversible." = "En supprimant votre compte, toutes les notes créées avec ce compte seront supprimées définitivement. Cette action est irréversible.";
+"By deleting the account for %@, all notes created with this account will be permanently deleted. This action is not reversible." = "En supprimant le compte associé à %@, toutes les notes créées avec ce compte seront définitivement supprimées. Cette action est irréversible.";
```

Comment:
I'm a bit surprised that there is a string copy change here, and more importantly one that changes the list of placeholders (from no placeholder, to expecting a `%@` placeholder), while there's no related change in code itself.

That might mean that we use this string somewhere in the code base, but before it didn't require a String (`%@`) parameter to be formatted but now it does, so to code might crash if we didn't provide one at call site…

---

I'm not sure how the _key_ could have changed in GlotPress anyway (or does that mean that we updated the original `.strings` file that GlotPress imports?… but I don't see such change in this diff), this is quite suspicious tbh…

Reply:
@oguzkocer @AliSoftware I think this is a side effect of the 2.16 ad-hoc code freeze.

The code that changed that string went in [in this commit](https://github.com/Automattic/simplenote-macos/commit/2d4ff421989e7da7f90665f3eb598bc66cc25e05) (well done @charliescheer for keeping it small and easy to identify 💙 ). Not relevant to the outcome but worth pointing out that [the PR that commit is from](https://github.com/Automattic/simplenote-macos/pull/1026) went straight into `release/2.15`, not `develop`.

When @oguzkocer run the 2.16 code freeze, the `en` `strings` file was update correctly, as you can see [here](https://github.com/Automattic/simplenote-macos/commit/fb76f1020895a42404c00e0d9caee9544ad7684a) (I wish our automation appended the release version to the commit message, but you can verify that's for version 2.16 in different ways, such as looking at the tags reported by GitHub or by exploring the files for that SHA1).

I haven't verified it with @oguzkocer, but I'm pretty sure that, because 2.16 was an ad hoc release with the only purpose of letting us update the name on App Store Connect, it was created and finalized in one go. I'm saying this because the [first commit](https://github.com/Automattic/simplenote-macos/commit/a19ffeb7eb0a87aed1b22e9cb4a2fc87c638dff3) and [the last](https://github.com/Automattic/simplenote-macos/commit/34e368f5fcb7fa9aab4c4fd48e106d40136c10fb) are ~20 minutes apart. 

The 2.16 work uploaded the new `en` strings to GlotPress, but didn't have the time to fetch the translated updates. Now that we finalized 2.17, those values have come through.

Reply:
Correct, we did a code freeze and finalize release at the same time in both `2.16` and `2.17`. The goal was to keep our regular flow consistent and avoid any mistakes we might make by manually changing things.

This all makes sense to me, thank you for the clarification @mokagio 🙇‍♂️ I guess that means these PRs are good to merge and I can release the binary in App Store. (which I'll do now to keep things moving)

Reply:
Hey y'all.  Just to give a little context to what happened.  There was some confusion over the wording for the delete account alert.  This conversation came up during beta testing and it was decided to update the language before releasing the delete account feature.  You can see that conversation at this internal ref: p5T066-2Co-p2#comment-9776

It was requested that we get those changes out  before the release, so it went straight into the release branch instead of into dev.

Reply:
That all makes sense now, and this means that we will actually fix an issue with this 2.17 as we will use the new copy (introduced in the release branch of 2.16) for all the translations now, instead of the new copy for English but old one for other locales (well I guess since the key changed, that means non-English locales used the new English copy untranslated in 2.16 in practice). So this is a good change after all! 🎉

Reply:
PS: Great investigation work @mokagio 👍 👏 🕵️